discovery-provider/plugins/verified-uploads/src/handlers/tracks.js

```diff
@@ -0,0 +1,54 @@
+import { dp_db } from "../db.js";
+import { slack } from "../slack.js";
+import dotenv from "dotenv";
+
+dotenv.config();
+const { TRACKS_SLACK_CHANNEL } = process.env;
+
+export default async ({ track_id }) => {
+  const trackId = track_id;
+  const results = await dp_db("tracks")
+    .innerJoin("users", "tracks.owner_id", "=", "users.user_id")
+    .innerJoin("track_routes", "tracks.track_id", "=", "track_routes.track_id")
+    .select(
+      "tracks.title",
+      "tracks.mood",
+      "tracks.genre",
+      "tracks.release_date",
+      "tracks.is_premium",
+      "tracks.owner_id",
+      "users.user_id",
+      "users.handle",
+      "users.name",
+      "tracks.track_id",
+      "users.is_verified",
+      "track_routes.slug",
+      "tracks.created_at",
+      "tracks.updated_at"
+    )
+    .where("tracks.track_id", "=", trackId)
+    .where("users.is_verified", "=", true)
```

Comment:
do we need to check tracks.is_current = true or no?

Reply:
on the user or tracks? i think it's possible this is false if we query and another modification comes in between the time the PG notify occurs and the query executes, may be best to leave out?

Reply:
on tracks, i trust your judgement tho!

Reply:
Actually I think I misspoke, I'm pulling this field directly from the message (latest row) so it may always be true anyways?

Reply:
yea good point, it may not hurt to double check by filtering just in case, but totally up to you idk the full flow well enough to know whats needed lol